system/kustomize/metal-operator-managedresources/rbac.yaml

```diff
@@ -1,145 +0,0 @@
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: cc:oidc-ias-admin
```

Comment:
All the `cc:oidc*` clusterrolebindings are missing in the managed resources right now, no?